frontend/src/component/providers/AccessProvider/AccessProvider.tsx

```diff
@@ -70,22 +70,12 @@ const checkPermission = (
     if (
         p.permission === permission &&
         (p.project === project || p.project === '*') &&
-        (p.environment === environment || p.environment === '*')
+        (!p.environment ||
+            p.environment === environment ||
+            p.environment === '*')
```

Comment:
Unrelated slight refactor from the 2 conditions we had before:

```TS
if (
    p.permission === permission &&
    (p.project === project || p.project === '*') &&
    (p.environment === environment || p.environment === '*')
) {
    return true;
}

if (
    p.permission === permission &&
    (p.project === project || p.project === '*') &&
    !Boolean(p.environment)
) {
    return true;
}
```

Should achieve the same end result.